doc/faq.md

```diff
@@ -42,3 +42,119 @@ Second, edit the assessment `pl-exp101/courseInstance/Fa17/assessments/final/inf
+
+### Why is my QID invalid?
+
+QID stands for the *Q**uestion **Id**entifier given to the folder that contains
+the question information -- e.g. `info.json`, `question.html`, and `server.py` 
+(optional) -- in `questions/`. QIDs can be invalid if they are spelled incorrectly
+or simply are not present in the directory. On sync, these issues are picked up
+and displayed as an error.
+
+```bash
+Error: invalid QID: "question_name"
+```
+
+To resolve this issue, first check the name of the folder inside of `questions/`
+and, then, check the question name listed in `infoAssessments.json`. Either
+rename the folder or change the name listed in assessments to be the same.
+
+See [Directory Structure](question.md#directory-structure) for more details.
+
+### Why do I have a Syntax Error in my JSON file?
+
+During the sync process, all `.json` files are validated. If any syntax issues
+arise, then an error message will be triggered. 
+
+The most common error is there is a missing a comma after the prior entry
+in a `infoAssessment.json` file. As a result, an unexpect token would be found.
+
+```bash
+Error: Error in JSON file format: file.json (line 55, column 17)
+
+SyntaxError: Unexpected token '{' at 55:17
+                {"id": "question_name", "points": [2, 1]},
+```
+				
+For example, this error would be triggered under:
+
+```json
+"zones": [
+    {
+        "title": "Easy questions",
+        "questions": [
+            {"id": "anEasyQ", "points": [10, 5, 3, 1, 0.5, 0.25]},
+            {"id": "aSlightlyHarderQ", "points": [10, 9, 7, 5]}
+        ]
+    },
+    {
+        "title": "Hard questions",
+        "questions": [
+            {"id": "hardQV1", "points": 10}     # <----- No comma, but another question
+            {"id": "question_name", "points": [2, 1]},
+            {
+                "numberChoose": 1,
+                "points": 5,
+                "alternatives": [
+                    {"id": "FirstAltQ", "points": 10},
+                    {"id": "SecondAltQ"}
+                ]
+            }
+        ]
+    }
+],
+```
+
+See [Question Specification](assessment.md#question-specification) for more details.
+
+### Why is the UUID used in multiple questions?
+
+While writing PrairieLearn questions, it is very common to copy an existing
+question and slightly modify for new purposes. Often times, we forget to update
+the `uuid` that is in the `info.json` file associated with a `v3` question.
+As a result, we'll end up receiving a duplication error when we try to sync
+the question base to the website.
+
+```bash
+<<<<<<< Local Changes
+Error message: Error: UUID 7a323f4c-cafd-40b2-8576-7db18e3f439b is used in multiple questions: Q1, Q2
+=======
+Error message: Error: UUID 7a323f4c-cafd-40b2-8576-7db18e3f439b is used in multiple questions: question_name_uuid_original, question_name_uuid_duplicate
+>>>>>>> External Changes
+```
```

Comment:
Is the text above output from a `git diff`? I'm not sure why we have the `<<<<<` stuff. I'm also not sure why we have two lines, one with `Q1, Q2` and one with `question_name_uuid_original, question_name_uuid_duplicate`.